client/src/services/smallDisplayWarning.ts

```diff
@@ -0,0 +1,47 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { Information, InformationLevel, InformationManager, PresentationMode } from '@/services/informationManager';
+import { useWindowSize } from 'megashark-lib';
+import { onMounted, onUnmounted, watch } from 'vue';
+
+// Global, will only reset on page refresh
+let warnedAboutDisplay = false;
+
+export function useSmallDisplayWarning(infoManager?: InformationManager): any {
```

Comment:
Maybe add doc to explain how it works?
```suggestion
/** Display warning dialog on small screens
 *
 * The warning is shown only once per refresh and it is also shown if the user
 * reduces the size of the window. 
 */
export function useSmallDisplayWarning(infoManager?: InformationManager): any {
```